subs:
  ros__parameters:
    # cam_topics: ['right_camera/image', 'left_camera/image']
    cam_topics: ['right_camera/image']
    model_path: '/home/chengjing/Desktop/multi_cam_detection/src/multi_cam_obj_detection/multi_cam_obj_detection/model'
    confidence_threshold: 0.5
    queue_size : 5
    slop: 0.1